utils/build-presets.ini

```diff
@@ -2119,19 +2118,11 @@ darwin-toolchain-installer-package=%(darwin_toolchain_installer_package)s
 
 [preset: tensorflow_linux]
 mixin-preset=buildbot_linux
-legacy-impl
-indexstore-db=0
```

Comment:
Did you try building a toolchain to verify whether these `=0` lines can be removed?